src/index.html

```diff
@@ -1,12 +1,275 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.css">
-  </head>
-  <body>
-    <h1>Catalog</h1>
-  </body>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Catalog</title>
+  <link rel="stylesheet" href="./styles/main.css">
+</head>
+
+<body>
+  <header class="header" data-qa="card">
+    <nav class="header__nav nav">
+      <a class="header-logo__link">
+        <img class="nav__logo" alt="logotype of moyo" src="images/logo.png" width="40px" height="40px">
+      </a>
+      <ul class="nav__list">
+        <li class="nav__item nav__list__item--active">
+          <a href="#" class="nav__link is-active">Apple</a>
+        </li>
+        <li class="nav__item">
+          <a href="#" class="nav__link">Samsung</a>
+        </li>
+        <li class="nav__item">
+          <a href="#" class="nav__link">Смарфоны и телефоны</a>
+        </li>
+        <li class="nav__item">
+          <a href="#" class="nav__link hovered" data-qa="nav-hover">Ноутбуки и компьютеры</a>
+        </li>
+        <li class="nav__item">
+          <a href="#" class="nav__link">Гаджеты</a>
+        </li>
+        <li class="nav__item">
+          <a href="#" class="nav__link">Планшеты</a>
+        </li>
+        <li class="nav__item">
+          <a href="#" class="nav__link">ФОТО</a>
+        </li>
+        <li class="nav__item">
+          <a href="#" class="nav__link">ВИДЕО</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+
+  <div class="container">
+    <div class="card-wrapper" data-qa="card">
+      <div class="card-wrapper__img">
+        <img src="./images/imac.jpeg" alt="imac">
+      </div>
+      <div class="card-wrapper__discription">
+        Моноблок APPLE A1419 iMac
+        27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="card-wrapper__code">Код товара: 195434
+      </div>
+      <div class="rating">
+        <div class="rating__star--active"></div>
+        <div class="rating__star--active"></div>
+        <div class="rating__star--active"></div>
+        <div class="rating__star--active"></div>
```

Comment:
якщо є модифікатор має бути і сам клас
```suggestion
        <div class="rating__star rating__star--active"></div>
```

Reply:
?

Reply:
Я тут не розумію, як має бути: блок є( rating), елемент є (rating__star), модифікатор(стан) -теж є (rating__star--active)
який клас? куди його і навіщо?